aptos-move/aptos-gas-schedule/src/ver.rs

```diff
@@ -64,7 +66,7 @@
 ///       global operations.
 /// - V1
 ///   - TBA
-pub const LATEST_GAS_FEATURE_VERSION: u64 = 20;
+pub const LATEST_GAS_FEATURE_VERSION: u64 = gas_feature_versions::RELEASE_V1_16;
```

Comment:
actually, RELEASE_V1_16 does not exist yet

Reply:
you need to add it here first https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas-schedule/src/ver.rs

Reply:
it would be gas version 21

Reply:
I added a new version + description? Or do you mean just use 21?

Reply:
add new version and description, also need to add to main, not sure why we missed this for this branch, probably because there were no other gas change